### ISSUE TITLE
Feature Request: support step_backward in asciinema cli play command

### ISSUE THREAD

**Issue by xguo-prestolabs** [asciinema/asciinema#651]
Hi, all,

asciinema cli only support 4 key bindings, 
1. quit (Ctrl+C)
2. pause (SPACE)
3. step/step_forward (DOT)
4. next_marker ( ] )

I think we also need step_backward key, is there any reason for not adding support of this step_backward key?

Thanks.

**Comment by ku1ik** [comment]
> is there any reason for not adding support of this step_backward key?

Yup. The way `asciinema play` works doesn't allow rewinding / step back. Step forward is easy because we just need to write the next output event to STDOUT. Step backward would need to run a virtual terminal emulator inside your actual terminal to be able to obtain screen contents for each (past) frame, and then redrawing the whole terminal view. In other words, while step forward was trivial to implement, step backward is hard (possible but a lot of work).

**Comment by ku1ik** [comment]
Not planned at the moment.
